samples/ChatSample/ChatSample/Startup.cs

```diff
@@ -28,7 +28,7 @@ public void Configure(IApplicationBuilder app)
             app.UseAzureSignalR(routes =>
             {
                 routes.MapHub<Chat>("/chat");
-                routes.MapHub<BenchHub>("/bench");
+                routes.MapHub<EchoHub>("/signalrbench");

```

Comment:
>EchoHub [](start = 30, length = 7)

why change?

Reply:
What's this used for？
Our azure-signalr-bench requires a Hub with simple Echo implementation.
And it needs to be registered as "/signalrbench".

Reply:
benchHub is used for `azure-signalr-bench`

---
In reply to: [350527793](https://github.com/Azure/azure-signalr/pull/744#discussion_r350527793) [](ancestors = 350527793)

Reply:
Ok, I got it.
Since the URL has been updated to `/signalrbench` so I will just change this and reuse the `EchoHub`.